src/casp/services/db/models/cells.py

```diff
@@ -0,0 +1,78 @@
+import datetime
+
+import sqlalchemy as sa
+from sqlalchemy.dialects.postgresql import JSONB
+from sqlalchemy.orm import backref, relationship
+
+from casp.services import db
+
+
+class CellIngestInfo(db.Base):
+    id = sa.Column(sa.Integer, primary_key=True)
+    cas_ingest_id = sa.Column(sa.String(255), unique=True, nullable=False)
+    dataset_id = sa.Column(sa.String(255), nullable=False)
+    uns_metadata = sa.Column(JSONB, nullable=False, default={})
+    ingest_timestamp = sa.Column(sa.DateTime, default=datetime.datetime.utcnow)
+
+    __tablename__ = "cells_ingestinfo"
```

Comment:
In BQ, we are calling that `cas_ingest_info`. Why are we calling this `cells_ingestinfo`?